.github/workflows/scorecards.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # tag=v3.0.0
+        uses: actions/checkout@v3
```

Comment:
While I'm not generally a fan of this, the scorecard recommends using the commit hash rather than a tag.  The other actions in this workflow all use the hash; it'd be good to at least remain consistent with the file and potentially decide one way or another separately from this upgrade PR.

Reply:
I see, good point. I'll switch back to keep the commit format.